mechanics/RELEASE-SCHEDULE.md

```diff
@@ -11,12 +11,13 @@ With that said, it can be useful to have a list of when future releases will hap
 
 | Release | Date       | EOL        | Min K8s Version | Notes                         |
 | ------- | ---------- | ---------- | --------------- | ----------------------------- |
-| 0.23    | 2021-05-18 | 2021-11-02 | 1.18            | Removes v1beta1 Eventing APIs |
 | 0.24    | 2021-06-29 | 2021-12-14 | 1.19            | |
 | 0.25    | 2021-08-10 | 2022-01-25 | 1.19            | |
 | 0.26    | 2021-09-21 | 2022-03-08 | 1.19            | |
-| 0.27    | 2021-11-02 | 2022-04-19 | 1.20            | |
-| 0.28    | 2021-12-14 | 2022-05-31 | 1.20            | |
+| 1.0     | 2021-11-02 | 2022-04-19 | 1.20            | |
+| 1.1     | 2021-12-14 | 2022-05-31 | 1.20            | |
+| 1.2     | 2022-01-25 | 2022-06-12 | 1.20            | |
```

Comment:
1.20 goes into maintenance mode Dec 28th. We should target 1.21 for this release
Details: https://kubernetes.io/releases/patch-releases/#1-20

Reply:
Ah I was looking at EOL and not maintenance mode

Reply:
Kinda ambiguous on our versioning principles
> We (the community) support the range of upstream minor Kubernetes versions that are Kubernetes community supported at the time of the cut

Reply:
Yeah I need to update that